[Focus Mode] Update the overview and about-tools pages to match Focus Mode (#2668)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/overview.md

If the **Activity Bar** is located on the left side of the DevTools window, vertically, only the tool icons appear, and the tool names appear when you hover over the tool icons.

<!-- ------------------------------ -->
#### Change the location of the Activity Bar

By default, the **Activity Bar** is displayed horizontally at the top of the DevTools window. To maximize usable screen space in a variety of DevTools window size and locations, you can change the location of the **Activity Bar** to the left side of the DevTools window or to be automatically set depending on the DevTools window docking location.

The **Activity Bar** can be in one of the following locations:

* **Top** (default): The **Activity Bar** will always be horizontal.

* **Left side**: The **Activity Bar** will always be vertical.

* **Adapt to dock location**: The **Activity Bar** will be in a horizontal or vertical orientation depending on where DevTools is docked.

  * If DevTools is docked to the left or right side of the browser window, the **Activity Bar** will be vertical.

>   * If DevTools is docked to the bottom side of the browser window, or undocked in its own window, the **Activity Bar** will be horizontal.

Reviewer: ```suggestion
  * When DevTools is docked to the bottom side of the browser window, or undocked in its own window, the **Activity Bar** is horizontal.
```
prefer present tense per corp style guide
https://learn.microsoft.com/en-us/style-guide/grammar/verbs

   *  **Performance** tool.
   *  **Memory** tool.
   *  **Application** tool.

* Icons:
   *  **Move Activity Bar to left** (![Move Activity Bar to left icon](./overview-images/move-activity-bar-to-left-icon.png)) button.
   *  **More Tools** (![More Tools icon](./overview-images/more-tools-icon-light-theme.png)) button.
   *  **Customize and control DevTools** (![Customize icon](./overview-images/customize-devtools-icon-light-theme.png)) menu button.
   *  **Help** (![Help icon](./overview-images/help-icon.png)) button.
   *  **Close** DevTools (![Close DevTools icon](./overview-images/close-devtools-icon-light-theme.png)) button.

![The main toolbar, with labels that identify its features](./overview-images/devtools-intro-menu-bar.png)

The **Activity Bar** features are described below.

<!-- ------------------------------ -->
#### Pin and rearrange tools in the Activity Bar

You can choose which tools appear in the **Activity Bar** by pinning or unpinning tools. This allows you to customize DevTools to your preferred workflow.

> Click **More tools** (**+**) to list all the available tools. Selecting a tool will pin it to the **Activity Bar**. It will be shown by default every time DevTools is opened.

Reviewer: ```suggestion
Click **More tools** (**+**) to list all the available tools. Selecting a tool will pin it to the **Activity Bar**. The tool is then present in the Activity Bar whenever DevTools is opened.
```
* "it" - i avoid indirection pointer, unless that makes same word appear twice in a row.
* "by default" - might add unnecessary complexity; the main idea is that the tool stays there.

![The More tools menu in Focus Mode, with the expanded list of all tools](./overview-images/more-tools.png)

> If there isn't enough space in the **Activity Bar** to display all pinned tools, some tools may overflow into the **More tools** menu:

Reviewer: i think this is misleading.  it sounds automatic.  who is the actor?  i think this never happens unless user actively moves.  either say "automatically" or "you can move..."

Author: It happens automatically when the user keeps adding more tools to the activity bar. At some point, there isn't enough space to fit them all in the activity bar, and only a few get displayed there, while the rest gets displayed into the More Tools menu.
